papermill/cli.py

```diff
@@ -97,19 +97,17 @@ def _resolve_type(value):
 
 
 def _is_int(value):
-    try:
-        int(value)
-    except ValueError:
+    if type(value) is int:
+        return True
+    else:
```

Comment:
Welp, I made a mistake. The reason this used `int` to see if a value was an int was that it was converting string parameters from the command line (since it only passes strings). The casting was a way to convert from string to int, float, etc.

I can either revert this _or_ you can make a new PR that puts this back in place with a comment about using casting for conversion...

Reply:
Ohh, I see what you mean. No worries, I'll put it back in place and add a comment

Reply:
So just to be clear, is it okay even if an int was converted to a float? Based on the old version, in the `_resolve_type` function, any int will be converted to a float and it will never reach the `_is_int` cast.

Reply:
Yeah that's fine, float should take primacy.